### DISCUSSION TITLE
Periodic Boundary Conditions

### DISCUSSION THREAD

**Discussion by PouriyaTUDelft** [tobac-project/tobac#280] [Q&A] [answered]
Hi to the developers,

I want to apply the code to track some features in LES models with double-periodic boundary conditions. Can tobac track the features in these simulations as well? and if so, in which function should I tell tobac to consider it?

In advance, thanks for the reply!

**Comment by freemansw1** [comment, marked as answer]
Hi @PouriyaTUDelft ! Great question. This is a capability that we are actively working on incorporating. We have a pull request in now to do exactly that (#127). It needs some refactoring and other cleaning up, but if you're interested, I encourage you to pull down that PR and give that code a test. If you decide to do so, let us know how it goes!

**Comment by PouriyaTUDelft** [comment]
thanks for the quick reply,
Will let you know once I try it.

**Comment by JuliaKukulies** [comment]
@PouriyaTUDelft, I hope it is OK that I converted your issue into a discussion. As highlighted by @freemansw1, this feature  is already implemented and will appear in the next version of tobac. It is, hence, not a formal task anymore, but since we think that you raised a great question that others will wonder about as well, we want to keep your question in the discussion!

**Reply by PouriyaTUDelft** [reply]
that's perfect!
